### ISSUE TITLE
Question: Can we have dash output with the result being N repetitions of the input file

### ISSUE THREAD

**Issue by parth2094** [google/shaka-packager#516]
## System info
**Operating System**: Ubuntu 14.04
**Shaka Packager Version**: 53aa775-release

## Issue and steps to reproduce the problem

**Packager Command**:
'in=/media/default.mp4,stream=audio,init_segment=/media/defaultInputResult/init.m4a,segment_template=/media/defaultInputResult/$Number$.m4a' --generate_static_mpd --segment_duration 10 --mpd_output /media/defaultInputResult/master.mpd

Extra steps to reproduce the problem?
(1) Followed the steps here : https://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html
to get shaka player up and running.
(2) Hand curated an mpd to have segment List(23 chunks) instead of segmentTemplate. I have attached the mpd file in txt format.
[mpd.txt](https://github.com/google/shaka-packager/files/2623375/mpd.txt)
(3) Played this mpd using shaka player and it works i.e it is able to play all 23 chunks generated by shaka.
(4) Now to test if the repetition of the same song can be played, I add chunk 24,25,26 to the segment list in mpd. Also created 24.m4a, 25.m4a, 26.m4a which are exact copies of 1.m4a, 2.m4a and 3.m4a respectively.
(5) Play the updated mpd via shaka player.

What is the expected result?
I expected that shaka player will play the newly added 3 chunks as well after the 23rd chunk

What happens instead?
Shaka player could only play till 23.m4a. 

There seems to be some information of the order in which the chunks must be played which is present either in the init or within each chunk in addition to the mpd file. If the order information was present only in mpd file then the player should have been able to play 24(copy of 1) after 23.m4a. Even though 24 is being fetched from the server, the shaka player isnt able to play it.

Also I tried 
"cat init.m4a 1.m4a 23.m4a 24.m4a > test.mp4"
 and played the test.m4 on vlc. Vlc was able to play all the chunks but with each chunk it had the original time of the chunk rather than relative time in test.m4. This means instead of time of 24.m4a being somewhere around 3 minutes, it shows its time as 0 sec. This further shows that there is some time/relative positioning information in init or the chunk itself because of which just adding an entry of the chunk into the mpd file is not enough to further play the added chunks.

In short, is there a shakapackager command or some tweaking of chunks/init/mpd that can help me create a resultant dash which is N repetitions of the input file?

<Please attach the input files or email to shaka-packager-issues@google.com.>
<Please provide any additional information below.>

**Comment by parth2094** [comment]
Thanks @kqyang , repeating the entire period worked in my case
